### DISCUSSION TITLE
Welcome to plausible-hugo Discussions!

### DISCUSSION THREAD

**Discussion by divinerites** [divinerites/plausible-hugo#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hpsof** [comment]
**thank you for providing this module!** 

plausible.io has a script 
- script.local.js | Allow analytics to track on localhost too which is useful in hybrid apps

one question:
- is there a possibilty to configure this in  [params.plausible]? 
debug = true does not work locally using _hugo serve_ ...
- so i had to deploy to my remote server (provider) to test working of the module
- am i missing something?

thank you

p.s. 
i use this locally 
`hugo.cmd serve --disableFastRender --noHTTPCache --cleanDestinationDir --bind 0.0.0.0 --baseURL 192.168.19.91`
